src/lib/sdc/network.sh

```diff
@@ -158,13 +158,13 @@ function valid_mtu
 
     if ! [[ $mtu =~ [1-9][0-9][0-9][0-9] ]] ; then
```

Comment:
this still hasn't been adjusted for the new maximum limit as Cody pointed out

Reply:
@jlevon I don't think that check actually does anything? Or, I just don't understand it?

```
Noah-Mehls-MacBook-Pro:~ noahmehl$ cat test.sh 
#!/bin/bash

    tag="overlay"
    mtu="$1"

    if ! [[ $mtu =~ [1-9][0-9][0-9][0-9] ]] ; then
        echo "Invalid mtu specified for tag $tag: $mtu - Test 1"
        echo "Valid MTU range is from 1500-65536"
        exit $SMF_EXIT_ERR_FATAL
    fi

    if [[ $mtu -gt 65536 || $mtu -lt 1500 ]]; then
        echo "Invalid mtu specified for tag $tag: $mtu - Test 2"
        echo "Valid MTU range is from 1500-65536"
        exit $SMF_EXIT_ERR_FATAL
    fi

Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh string
Invalid mtu specified for tag overlay: string - Test 1
Valid MTU range is from 1500-65536
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 1499
Invalid mtu specified for tag overlay: 1499 - Test 2
Valid MTU range is from 1500-65536
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 1500
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 9000
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 9001
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 65536
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 65537
Invalid mtu specified for tag overlay: 65537 - Test 2
Valid MTU range is from 1500-65536
```

Reply:
Sorry, to be clear, it will make sure it's a number, but it doesn't actually prevent 65337 from working...

Reply:
It's checking that it's an integer as you can see from your test. What's confusing now is that it's only checking the first four chars. Much better would be:

    if ! [[ $mtu =~ ^[0-9]{1,5}$ ]] ; then

Reply:
Sure, that's what the second check is for.

Reply:
@jlevon I agree, that is MUCH better:

```
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 6553.5
Invalid mtu specified for tag overlay: 6553.5 - Test 1
Valid MTU range is from 1500-65536
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 65537
Invalid mtu specified for tag overlay: 65537 - Test 2
Valid MTU range is from 1500-65536
Noah-Mehls-MacBook-Pro:~ noahmehl$ ./test.sh 655379
Invalid mtu specified for tag overlay: 655379 - Test 1
Valid MTU range is from 1500-65536
```

Reply:
I updated the PR for that :). @jlevon thanks for your BASH-foo :)